server/command/main.go

```diff
@@ -64,5 +64,5 @@ func (s *service) executeDebugLocations(params *params) (*model.CommandResponse,
 		return normalOut(params, md.MD("error"), err)
 	}
 	bytes, _ := json.Marshal(locations)
-	return normalOut(params, md.MD(string(bytes)), nil)
+	return normalOut(params, md.JSONBlock(string(bytes)), nil)
```

Comment:
You should just have to do `JSONBlock(locations)`, and it will format it properly with JSON indenting
```suggestion
	return normalOut(params, md.JSONBlock(string(bytes)), nil)
```